core/src/main/scala/chisel3/RawModule.scala

```diff
@@ -135,6 +135,9 @@ trait RequireSyncReset extends Module {
   override private[chisel3] def mkReset: Bool = Bool()
 }
 
+/** Mix with a [[RawModule]] to automatically connect DontCare to the module's ports, wires, and children instance IOs. */
+trait ImplicitInvalidate
```

Comment:
```suggestion
trait ImplicitInvalidate { self: RawModule => }
```
Maybe a self-type constraint?